### ISSUE TITLE
OpenShift Virtualization testing—End-User Documentation in nerc-docs repo

### ISSUE THREAD

**Issue by jtriley** [nerc-project/operations#727]
## Motivation
We need full end-user documentation on launching and working with VMs on OpenShift before we go live 

## Completion Criteria
End-user documentation merged to https://github.com/nerc-project/operations/issues/724 and live on the website (https://nerc-project.github.io/nerc-docs/)

**Comment by joachimweyl** [comment]
@jtriley is this a duplicate of #709

**Comment by joachimweyl** [comment]
This is not a duplicate of #709 because 709 is to create the internal documentation. This is blocked by 709 as this is Milson's work to get the documentation customer-facing.
